packages/rancher-alerting-drivers/README.md

```diff
@@ -0,0 +1,10 @@
+# rancher-alerting-drivers
+
+To add a new sub chart:
```

Comment:
Should this belong in the README.md? It wouldn't make sense to anyone who was installing the chart viewing this from the Rancher UI, so I think it may make sense to just copy what exists in the app-README.md

Reply:
oh, this supposes to be the readme for developers. 
Should I rename it to dev-README.md?

Reply:
sure that works. We still need a README.md as well I think.